tests/integration/src/test/scala/mesosphere/marathon/integration/setup/MesosTest.scala

```diff
@@ -372,7 +377,7 @@ trait MesosClusterTest extends Suite with ZookeeperServerTest with MesosTest wit
     mesosMasterUrl,
     autoStart = false,
     config = mesosConfig,
-    waitForMesosTimeout = patienceConfig.timeout.toMillis.milliseconds
+    waitForMesosTimeout = patienceConfig.timeout.toMillis.milliseconds - 10.seconds
```

Comment:
I'd rather just create a separate config. Right now, we would wait for Mesos `5min-10s`? Which is still way too much - Mesos master/agents are usually started within a few seconds and definitely <1m